### PR TITLE
sql: fix some format mistakes

### PR DESCRIPTION
**Pull request by IzabelWang**
Fix some format mistakes in op-guide\tidb-config-file.md and sql\slow-query.md.
PTAL @lilin90 @bb7133

### CODE REVIEW

File: sql/slow-query.md [hunk 1]
```diff
@@ -157,17 +157,17 @@ tidb > select query, query_time, stats from INFORMATION_SCHEMA.`SLOW_QUERY` wher
...
+| select * from t1 where a>2; | 0.602011247 | t1: pseudo                       |
+| select * from t1 where a>3; | 0.50077719  | t1: pseudo                       |
+| select * from t1 join t2;   | 0.931260518 | t1: 407872303825682445,t2: pseudo |
 +-----------------------------+-------------+---------------------------------+
 ```
 
 #### 解析其他的 TiDB 慢日志文件
 
-目前查询 `INFORMATION_SCHEMA.SLOW_QUERY` 只会解析配置文件中 `slow-query-file` 设置的慢日志文件名，默认是 "tidb-slow.log" 。 但如果想要解析其他的日志文件，可以通过设置 session 变量 `tidb_slow_query_file` 为具体的文件路径，然后查询 `INFORMATION_SCHEMA.SLOW_QUERY` 就会按照设置的路径去解析慢日志文件。
+目前查询 `INFORMATION_SCHEMA.SLOW_QUERY` 只会解析配置文件中 `slow-query-file` 设置的慢日志文件名，默认是 "tidb-slow.log"。 但如果想要解析其他的日志文件，可以通过设置 session 变量 `tidb_slow_query_file` 为具体的文件路径，然后查询 `INFORMATION_SCHEMA.SLOW_QUERY` 就会按照设置的路径去解析慢日志文件。
```
Comment (lilin90) [inline sql/slow-query.md]:
```suggestion
目前查询 `INFORMATION_SCHEMA.SLOW_QUERY` 只会解析配置文件中 `slow-query-file` 设置的慢日志文件名，默认是 "tidb-slow.log"。但如果想要解析其他的日志文件，可以通过设置 session 变量 `tidb_slow_query_file` 为具体的文件路径，然后查询 `INFORMATION_SCHEMA.SLOW_QUERY` 就会按照设置的路径去解析慢日志文件。
```

File: sql/slow-query.md [hunk 2]
```diff
@@ -157,17 +157,17 @@ tidb > select query, query_time, stats from INFORMATION_SCHEMA.`SLOW_QUERY` wher
 +-----------------------------+-------------+---------------------------------+
 | query                       | query_time  | stats                           |
 +-----------------------------+-------------+---------------------------------+
-| select * from t1 where a=1; | 0.302558006 | t1:pseudo                       |
-| select * from t1 where a=2; | 0.401313532 | t1:pseudo                       |
-| select * from t1 where a>2; | 0.602011247 | t1:pseudo                       |
-| select * from t1 where a>3; | 0.50077719  | t1:pseudo                       |
-| select * from t1 join t2;   | 0.931260518 | t1:407872303825682445,t2:pseudo |
+| select * from t1 where a=1; | 0.302558006 | t1: pseudo                       |
```
Comment (lilin90) [inline sql/slow-query.md]: These spaces in this code block might be unnecessary. Please check with @crazycs520.

Comment (crazycs520) [reply sql/slow-query.md]: Yes, It is the result of the sql query.

Comment (lilin90) [reply sql/slow-query.md]: @IzabelWang So please change it back.

File: op-guide/tidb-config-file.md [hunk 3]
```diff
@@ -47,68 +47,68 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
-+ 默认: 300ms
++ 默认：300ms
 + 当查询大于这个值，就会当做是一个慢查询，输出到慢查询日志。
 
 ### `expensive-threshold`
 
 + 输出 `expensive` 操作的行数阈值。
-+ 默认: 10000
++ 默认：10000
 + 当查询的行数(包括中间结果，基于统计信息)大于这个值，我们就会当成是一个 `expensive` 的操作，输出一个前缀带有 `[EXPENSIVE_QUERY]` 的日志。
```
Comment (bb7133) [inline op-guide/tidb-config-file.md]:
```suggestion
+ 当查询的行数（包括中间结果，基于统计信息）大于这个值，我们就会当成是一个 `expensive` 的操作，输出一个前缀带有 `[EXPENSIVE_QUERY]` 的日志。
```

File: sql/slow-query.md [hunk 4]
```diff
@@ -48,9 +48,9 @@ select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
 * `Process_keys`：表示 Coprocessor 处理的 key 的数量。相比 total_keys，processed_keys 不包含 MVCC 的旧版本。如果 processed_keys 和 total_keys 相差很大，说明旧版本比较多。
 * `DB`：表示当前的 database。
 * `Index_ids`：表示语句涉及到的索引的 ID
```
Comment (bb7133) [inline sql/slow-query.md]:
```suggestion
* `Index_ids`：表示语句涉及到的索引的 ID。
```

File: sql/slow-query.md [hunk 5]
```diff
@@ -48,9 +48,9 @@ select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
...
-* `Is_internal`：表示是否是 TiDB 内部 SQL。true 为TiDB 内部执行的SQL, 比如 analyze，load variable 等；false 为用户执行的 SQL 。
+* `Is_internal`：表示是否是 TiDB 内部 SQL。true 为TiDB 内部执行的SQL，比如 analyze，load variable 等；false 为用户执行的 SQL。
 * `Digest`：表示 SQL 语句的指纹。
-* `Memory_max`：表示执行期间做多时候使用的内存数量, 单位为byte。
+* `Memory_max`：表示执行期间做多时候使用的内存数量，单位为byte。
```
Comment (bb7133) [inline sql/slow-query.md]:
```suggestion
* `Memory_max`：表示执行期间做多时候使用的内存数量，单位为 byte。
```

File: sql/slow-query.md [hunk 6]
```diff
@@ -48,9 +48,9 @@ select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
...
 * `Num_cop_tasks`：表示 cop-tasks 的数目。
 * `Cop_proc_avg`：cop-task 的平均执行时间。
 * `Cop_proc_p90`：cop-task 的P90分位执行时间。
```
Comment (bb7133) [inline sql/slow-query.md]:
```suggestion
* `Cop_proc_p90`：cop-task 的 P90 分位执行时间。
```

File: sql/slow-query.md (hunk 2 above)
Comment (IzabelWang) [reply sql/slow-query.md]: OK, done~

File: sql/slow-query.md [hunk 7]
```diff
@@ -161,13 +161,13 @@ tidb > select query, query_time, stats from INFORMATION_SCHEMA.`SLOW_QUERY` wher
 | select * from t1 where a=2; | 0.401313532 | t1:pseudo                       |
 | select * from t1 where a>2; | 0.602011247 | t1:pseudo                       |
 | select * from t1 where a>3; | 0.50077719  | t1:pseudo                       |
-| select * from t1 join t2;   | 0.931260518 | t1:407872303825682445,t2:pseudo |
+| select * from t1 join t2;   | 0.931260518 | t1:407872303825682445,t2: pseudo |
```
Comment (lilin90) [inline sql/slow-query.md]: Why leave one space for this line?

Comment (IzabelWang) [reply sql/slow-query.md]: Sorry & addressed~

**Pull request merged**